taxonomies/food/ingredients.txt

```diff
@@ -10450,6 +10450,7 @@ fr:concentré salin liquide
 <en:salt
 en:black salt
 bg:черна сол
+de:schwarzes Salz
```

Comment:
Looks like this is [Kala Namak](https://www.wikidata.org/wiki/Q1572414)? It seems like that Name, or even "Schwarzsalz" is the more common variant? (Personally, I don't know it)

Reply:
Yes it seems to be the same, I added the two synonyms. I get more results for schwarzes Salz than the 2 other synonymes on google, it would be interesting to know which one of the 3 versions is the more common on the ingredient lists of OFF.